ui/src/main/resources/CKEditor/Translations.xml

```diff
@@ -58,7 +58,7 @@ ckeditor.admin.reset.confirmation=Are you sure you want to restore the default c
 ckeditor.plugin.link.options=Options
 ckeditor.plugin.link.anchor=Anchor
 ckeditor.plugin.link.queryString=Query String
-ckeditor.plugin.link.defaultLabel=type the link label
+ckeditor.plugin.link.defaultLabel=Type the link label
```

Comment:
Question B: do we need to provide a default value? or can we just use a placeholder for the input? (//depends on the answer to A, since if we add a link without a value, it will be hard for the user to know where the link starts/ends). 
* A solution would be to make the "Display Text" mandatory, but this will break cases when link display the page name if no link value is provided.